src/components/Pages/Home/Home.vue

```diff
@@ -1,11 +1,17 @@
 <template>
   <div>
-    <GetStartedBanner></GetStartedBanner>
+    <h2>Make your structured data available in our cloud</h2>
+    <p>If you've wanted to run your own Wikibase but don't have the infrastructure and support you'd need, Wikibase.cloud is for you. You provide the data – we provide a full-fledged and fully managed Wikibase. </p>
+    <h3>Coming soon</h3>
+    <p>Wikibase.cloud is in a closed beta, which means we're not yet accepting account requests. While you wait, consider joining the Wikibase community <a href="https://lists.wikimedia.org/postorius/lists/wikibase-cloud.lists.wikimedia.org/">mailing list</a>.</p>
+
+    <h3>Still have questions?</h3>
+    <p>Drop us a line using our <a href="https://wikiba.se/contact/">contact form</a>. Now is also a great time to get involved in the <a href="https://meta.wikimedia.org/wiki/Wikibase_Community_User_Group">Wikibase community</a>. </p>
   </div>
 </template>
 
 <script>
-import GetStartedBanner from '~/components/Pages/Home/LoggedOutFeatures/GetStartedBanner'
+// import GetStartedBanner from '~/components/Pages/Home/LoggedOutFeatures/GetStartedBanner'
 // import WikiCount from '~/components/Pages/Home/LoggedOutFeatures/WikiCount'
 // import WhatCanYouDo from '~/components/Pages/Home/LoggedOutFeatures/WhatCanYouDo'
 // import WhatIsIncluded from '~/components/Pages/Home/LoggedOutFeatures/WhatIsIncluded'
```

Comment:
All this old home stuff seems very outdated and unused by now, I think we can just remove it from the repo. Could also do that in a follow-up. What do you think?

Reply:
I will remove them anw if you said so. I don't think we also need those files anw so I'll delete them too